cmd/bcpd/app/codec.proto

```diff
@@ -13,7 +14,20 @@ import "github.com/iov-one/weave/x/validators/codec.proto";
 // Tx contains the message
 message Tx {
   // msg is a sum type over all allowed messages on this chain.
-  oneof sum{
+  Sum sum = 1;
```

Comment:
Is `sum` a good name for this field? Why not `message` anaolue to `BatchMsg`?

Reply:
It's not, it's just what we had before